Cargo.toml

```diff
@@ -18,6 +18,7 @@ lto = "thin"
 gateway-api = "0.15"
 http = "1"
 hyper = "1"
+hyper-util = { version = "0.1", default-features = false, features = ["tracing"] }
```

Comment:
I'd suggest that we avoid the workspace dep for now, since it's only needed at the runtime level for now.